src/OSPSuite.Core/Domain/ISimulation.cs

```diff
@@ -9,5 +10,7 @@ public interface ISimulation : ILazyLoadable, IAnalysable, IUsesObservedData, IM
       IEnumerable<CurveChart> Charts { get; }
 
       OutputMappings OutputMappings { get; set; }
+
+      DataRepository ResultsDataRepository { get; set; }
```

Comment:
Hum...this will require some explanation here. Why do we suddenly need some results in the ISimulation interface
This will create problems with implementation that have results already

Reply:
I agree a comment needs to be added - about the implementation, this actually already exists in MoBi Simulation, I have just added it to PkSimSimulation

Reply:
As I said, we already have it in PKSim simulation

My question is why do we need this. You can imagine that if the results were not defined in the ISimulation interface, there was probably a reason. Maybe because a Popsimulation also implements the ISim and it does not make sense? I don't have access to my laptop but I think this needs to be explained (e.g. To me;-)) because I doubt that this is needed

Reply:
I will check that there are no conflicts in PK-Sim and see if it is possible to avoid having it at all. ( btw I am also in vacation, so I will probably not be able to do any changes until the 31st of August )